exe/Daemon.hs

```diff
@@ -438,8 +438,8 @@ installMachineMessageHandler m handleMessage = do
 writeInputs :: MachineId -> [Value] -> Maybe Text -> Daemon [Value]
 writeInputs id inputs nonce = do
     (rs, idx) <- modifyMachine id $ \machine -> do
-        newIndex <- machineIpfs id $ writeIpfs id inputs
         (machine', results) <- addInputs inputs newIndex machine
+        newIndex <- machineIpfs id $ writeIpfs id inputs
```

Comment:
This does not work because we need to known the new index. Maybe it would make sense to split `writeIpfs` in two parts: One that just writes the new inputs to IPFS but does not update the IPNS record. The second part updates the IPNS record.

Reply:
Why write bad inputs to IPFS? Can we go back to the implementation before https://github.com/radicle-dev/radicle/commit/24db69a36fff7b58bd4425edb1b6824af42ff8a3?

Reply:
That would work, too